### DISCUSSION TITLE
Collect the types of indicators that people care about in venus components

### DISCUSSION THREAD

**Discussion by hunjixin** [filecoin-project/venus#4671] [Ideas]
We plan to start monitoring metrics next year. If anyone have information care about, you can list it in this discussion and give priority to support. Not only venus, but all the components in venus project.  write them separately according to the component type.

## venus

- [ ] time for process tipset
- [ ] current tipset height

## venus-messager

- [ ] time for waiting stable tipset
- [ ] time for message selection
- [ ] time for flusing data to database

## venus-gateway

- [ ] how may wallet are connecting
- [ ] how may sealer are ready for calculating  proof 

## venus-sealer/venus-worker

venus-sealer maybe deprerated next year, but can also list here, we will do in venus-cluster

**Comment by Fatman13** [comment]
From this [discussion](https://filecoinproject.slack.com/archives/C028PCH8L31/p1641304446278000) thread on slack.
